### ISSUE TITLE
SyncThing proof-of-concept

### ISSUE THREAD

**Issue by leigh-johnson** [bitsy-ai/printnanny-os#77]
**Describe your Request**

- [x] Add SyncThing client to PrintNanny OS image
- [x] Document how to install SyncThing client to desktop + exchange device ids

**Comment by leigh-johnson** [comment]
🎉 A preview is available in the v0.4 nightly. I'll take this for a spin tomorrow and write basic docs.

```
Build Configuration:
BB_VERSION           = "2.0.0"
BUILD_SYS            = "x86_64-linux"
NATIVELSBSTRING      = "ubuntu-22.04"
TARGET_SYS           = "aarch64-bitsy-linux"
MACHINE              = "raspberrypi4-64"
DISTRO               = "printnanny"
DISTRO_VERSION       = "0.4.0"
TUNE_FEATURES        = "aarch64 armv8a crc cortexa72"
TARGET_FPU           = ""
meta
meta-poky
meta-yocto-bsp       = "kirkstone:387ab5f18b17c3af3e9e30dc58584641a70f359f"
meta-raspberrypi     = "master:62a84833d90dfc18f8057bc5e99e96eff6798f12"
meta-oe
meta-python
meta-multimedia      = "master:a9e6d16e6640d105ecf3470ab891598034c9ef33"
meta-bitsy           = "main:0ddf9a6f546e3b59d7063af3e527f133a3aefa29"
meta-neural-network  = "master:1d70eb6c0f6d5a7d5dc904ae988a7e152b52267e"
meta-printnanny      = "main:0ddf9a6f546e3b59d7063af3e527f133a3aefa29"
meta-networking
meta-filesystems
meta-initramfs
meta-webserver       = "master:a9e6d16e6640d105ecf3470ab891598034c9ef33"
meta-swupdate        = "master:554b97ac9970ccdb41408f2440cd48446534a7d7"
workspace            = "kirkstone:387ab5f18b17c3af3e9e30dc58584641a70f359f"
```

We're going to start by using the relay pools @ https://relays.syncthing.net/ - if this takes off, I'll set up a relay on an unlimited bandwidth VPS
https://docs.syncthing.net/users/relaying.html

We're also using the default discovery servers for now:
https://docs.syncthing.net/users/stdiscosrv.html

**Comment by leigh-johnson** [comment]
Woo, SyncThing will be available in the 0.4 release! :tada: 

Created #90 to track a few ideas for fleshing out the documentation. If you're trying out Syncthing, feel free to add ideas/questions!
